src/docs/product/issues/issue-details/performance-issues/large-render-blocking-asset/index.mdx

```diff
@@ -0,0 +1,42 @@
+---
+title: "Large Render Blocking Asset"
+sidebar_order: 30
+redirect_from:
+  - /product/issues/performance-issues/large-render-blocking-asset/
+description: "Learn more about large render blocking assets and how to diagnose and fix them."
+---
+
+<Include name="early-adopter-note.mdx" />
+
+A _Large Render Blocking Asset_ is a performance problem that happens when a large asset causes a delay in displaying the page content. For example, if a page includes a stylesheet, the browser will pause rendering of the page until it downloads and processes the entire stylesheet. If the stylesheet is large, the user may see a blank or unstyled screen for a long time. This problem occurs when applications do not split up asset files, load non-essential assets synchronously, or load assets too early.
+
+## Detection Criteria
+
+The detector for this performance issue looks for specific asset (resource) spans with the following criteria:
+
+- The span operation has to be one of either: `resource.link` or `resource.script`
+- The span's `resource.render_blocking_status` data property must have a value of `blocking` (added by the `@sentry/tracing` SDK as of version `7.38.0`) or be missing.
+- The span's `Encoded Body Size` data property has to be over `1MB` in size
+- The [FCP (First Contentful Paint)](/product/performance/web-vitals/#first-contentful-paint-fcp) of the transaction must be at least `2` seconds (with an upper bound of `10` seconds to eliminate outliers).
+- The span's duration must be at least `33%` of the FCP
+
+## Span Evidence
+
+The evidence for a Large Render Blocking Asset problem identifies four aspects:
+
+- Transaction name
+- Asset URL
+- Effect on FCP
+- Effect on total transaction time
+
+View it by going to the Issues page in Sentry, selecting your project, clicking on the Large Render Blocking Asset error you want to examine, then scrolling down to the "Span Evidence" section in the "Details" tab.
+
+![Large Render Blocking Asset issue span evidence](span-evidence.png)
+
+## Fingerprinting
+
+Sentry computes the issue fingerprint based on the asset URL. We detect many different patterns of cache-busting hashes in URL paths, filenames, and query strings so that the same asset is grouped into a single issue even as its contents changes.
+
```

Comment:
```suggestion
Sentry computes the issue fingerprint based on the asset URL. We detect many different patterns of cache-busting hashes in URL paths, filenames, and query strings so that the same asset is grouped into a single issue even as its contents change.

```